src/views/About.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import styled from 'styled-components';
+import theme from '../theme/styledTheme';
+import luke from '../avatar/luke.jpg';
+import talent from '../avatar/talent.jpeg';
+import diana from '../avatar/diana.jpg';
+import mark from '../avatar/mark.jpg';
+import pavol from '../avatar/pavol.PNG';
+import connor from '../avatar/connor.jpg';
+
+class About extends React.Component {
+  render() {
+    return (  
+      
+      <PageOuter>
+         <h1 className="header">Meet The Team</h1>
+        <PageContainer>
+         
+        <section className="about-container">
+            <div className="card">
+              <img src={luke} alt=""/>
+               <h2>Lukasz Busk</h2>
+               <p>Project Manager</p>
+               <a href="https://github.com/Velsu"><i class="fab fa-github gh"></i></a>
+               <a href="https://uk.linkedin.com/"><i class="fab fa-linkedin tw"></i></a>
+               <a href="https://twitter.com"><i class="fab fa-twitter tw "></i></a>
+               </div>  
+               <div className="card">
+              <img src={diana} alt="Diana"/>
+               <h2>Diana Ignatescu</h2>
+               <p>Full Stack Developer</p>
+               <a href="https://github.com/DianaIgnatescu"><i class="fab fa-github gh"></i></a>
+               <a href="https://uk.linkedin.com/"><i class="fab fa-linkedin tw"></i></a>
```

Comment:
If you're gonna use linkedin if might be better to link to actual linkedin pages.